src/js_native_api_types.h

```diff
@@ -1,8 +1,7 @@
 #ifndef SRC_JS_NATIVE_API_TYPES_H_
 #define SRC_JS_NATIVE_API_TYPES_H_
 
-#include <stddef.h>
-#include <stdint.h>
+#include <cstdint>
```

Comment:
> Any idea why can't find cstdint in test ?

Because this is a C header file, not a C++ one :) So we should keep `stdint.h`.

Reply:
Can we use `#ifdef __cplusplus` in this case ?

Reply:
Yes, although I’d prefer `#include <stdint.h>` over
```
#ifdef __cplusplus
#include <cstdint>
#else
#include <stdint.h>
#endif
```

Reply:
I will change it to `#include <stdint.h> //NOLINT ` tormorrow :)

Reply:
Done.